src/kontrol/kdist/cheatcodes.md

```diff
@@ -77,6 +77,12 @@ module FOUNDRY-CHEAT-CODES
           <addressSet> .Set </addressSet>
           <storageSlotSet> .Set </storageSlotSet>
         </whitelist>
+        <mockCalls>
```

Comment:
@lisandrasilva  One other thing I forgot to point out. Above this configuration there is a short description of each cell and its role. Could you update that as well?

Reply:
Sure :)

Reply:
Thank you!